### PR TITLE
Get Camel lsp in Emacs blog

### PR DESCRIPTION
**Pull request by Navyashree0923**

### CODE REVIEW

File: content/blog/2022/06/index.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+ ;; If there is more than one, they won't work right.
+ )
+ 
+(require 'lsp-mode)
+(add-hook 'nxml-mode-hook #'lsp)
+```
+
+* Create `~/.emacs.d/lsp-camel.el` 
+
+In a `~/.emacs.d/lsp-camel.el` file:
```
Comment (apupier) [inline content/blog/2022/06/index.md:56]:
```suggestion
* Create `~/.emacs.d/lsp-camel.el`file with the following content:
```

File: content/blog/2022/06/index.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+(provide 'lsp-camel)
+;;; lsp-camel.el ends here
+```
+
+* Evaluating it by calling `M-x eval-buffer`
+
+* Loading Camel LS by calling: `M-x lsp-install-server` and choosing `camells`
+
+
+Follow this link to know more about configuring Emacs - (https://github.com/camel-tooling/camel-lsp-client-emacs)
```
Comment (apupier) [inline content/blog/2022/06/index.md:134]:
```suggestion
Follow [this link]((https://github.com/camel-tooling/camel-lsp-client-emacs) to know more about configuring Emacs and have up-to-date information. 
```

File: content/blog/2022/06/index.md [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+
+Then you can enjoy editing capabilities of Camel URI.
+
+NOTE - The configuration will be simplified with next release of lsp-mode
```
Comment (apupier) [inline content/blog/2022/06/index.md:138]:
```suggestion
NOTE - The configuration will be simplified with next release of [lsp-mode](https://github.com/emacs-lsp/lsp-mode) thanks to [built-in support of Camel Language Server](https://github.com/emacs-lsp/lsp-mode/issues/3528).
```

**Review by apupier — CHANGES_REQUESTED** [review]

**Comment by Navyashree0923** [comment]
Hello @apupier
I have updated the changes, please can you review it.

File: content/blog/2022/06/index.md [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+ )
+ 
+(require 'lsp-mode)
+(add-hook 'nxml-mode-hook #'lsp)
+```
+
+* Create `~/.emacs.d/lsp-camel.el`file with the following content:
+
+
+In a `~/.emacs.d/lsp-camel.el` file:
```
Comment (apupier) [inline content/blog/2022/06/index.md:59]:
```suggestion
```

File: content/blog/2022/06/index.md [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+title: "Client implementation of the Apache Camel Language Server Protocol for Emacs"
+date: 2022-06-08
+authors: [Navyashree0923]
+categories: ["Tooling"]
+preview: "Client implementation of the Apache Camel Language Server Protocol for Emacs"
+---
+
+# Text Editing capabilities of Camel URI with Camel XML DSL
+
+The capabilities are based on the Camel Language Server.
```
Comment (apupier) [inline content/blog/2022/06/index.md:11]:
```suggestion
The capabilities are based on the [Camel Language Server](https://github.com/camel-tooling/camel-language-server).
```

**Review by davsclaus — APPROVED** [review]

**Review by apupier — APPROVED** [review]
Thanks for this article that will surely interest Camel and emacs users.

**Comment by apupier** [comment]
build error is unrelated to this PR:

```
➤YN0000: [build:antora  ] [build:antora-perf] [21:01:42.971] ERROR (asciidoctor): target of include not found: Error-unused-starter-json.adoc
➤YN0000: [build:antora  ] [build:antora-perf]     file: docs/spring-boot/modules/ROOT/pages/list.adoc:19
➤YN0000: [build:antora  ] [build:antora-perf]     source: https://github.com/apache/camel-spring-boot.git (refname: main, start path: docs/spring-boot)
```

**Comment by apupier** [comment]
> maybe the "regen" commit was missing [apache/camel-spring-boot@43af0be](https://github.com/apache/camel-spring-boot/commit/43af0be0b8d1b61ac42c59ce9afc28e3d0c52d66) ? If it is the case, this other PR will be successful: #857

seems it was it as https://github.com/apache/camel-website/pull/857 was succesfull.

@Navyashree0923 Please update the date, I hope it will retrigger a build and the build should work

**Review by apupier — APPROVED** [review]

**Comment by zregvart** [comment]
@Navyashree0923 can you [edit your profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile#changing-your-profile-name) on GitHub to add **Name**, [the blog post](https://camel.apache.org/blog/2022/06/06/) currently shows only the picture from the profile without the name

**Pull request merged**